gui/src/app/state/psbt.rs

```diff
@@ -167,23 +168,15 @@ impl PsbtState {
                 return cmd;
             }
             Message::View(view::Message::Spend(view::SpendTxMessage::Broadcast)) => {
-                let outpoints: HashSet<_> = self.tx.coins.keys().cloned().collect();
+                let outpoints: Vec<_> = self.tx.coins.keys().cloned().collect();
                 return Command::perform(
                     async move {
                         daemon
-                            // TODO: filter for the outpoints in `tx.coins` when this is possible:
-                            // https://github.com/wizardsardine/liana/issues/677
-                            .list_coins()
+                            .list_coins(&[CoinStatus::Spending], &outpoints)
```

Comment:
we use the outpoint, it is enough, you can remove the status filter.

Reply:
I think it's OK to keep the status filter here as the previous `filter_map` was already filtering for coins with a non-null spend txid.

Reply:
Ah you may right, I think we are good then.